### PR TITLE
Update cli.py

### PR DESCRIPTION
**Pull request by lollozzo**
settings integration and custom methods

### CODE REVIEW

File: libetrv/cli.py [hunk 1]
```diff
@@ -74,6 +78,15 @@ def current_time(self):
     def set_setpoint(self, setpoint):
         self._device.temperature.set_point_temperature = setpoint
 
+    ## CUSTOM (TMP)
+    
+    def get_battery(self):
```
Comment (AdamStrojek) [inline libetrv/cli.py]: Can you reformat these functions by adding a single empty new line between them

File: libetrv/cli.py [hunk 2]
```diff
@@ -74,6 +78,15 @@ def current_time(self):
...
+        return self._device.battery
+    def get_name(self):
+        return self._device.name
+    def get_temperature(self):
+        temp = self._device.temperature
+        return str(temp.room_temperature) + "|" + str(temp.set_point_temperature)
```
Comment (AdamStrojek) [inline libetrv/cli.py]: Please use str.format for this

**Comment by boelle** [comment]
@lollozzo AdamStrojek requested some changes, not sure if you know this ?

**Comment by AdamStrojek** [comment]
Pull request still have issues that were not resolved

**Pull request closed without merging**